dist/sputility.js

```diff
@@ -1632,8 +1637,59 @@ var SPUtility = (function ($) {
 
    // get the display text of each resolved item and display in comma
    // delimited list
+   // updated: it will show as hyperlinks but not a plain text
    SPUserField2013.prototype.MakeReadOnly = function () {
-      return this._makeReadOnly(this._getValue());
+      var tmpArray = [];
+
+        $.each(this.GetValue(), function (key, val) {
+            if (val.Key !== null) { tmpArray.push(val.Key); }
+        });
+
+        var SPUserField = this;
+        var x = getUserId(tmpArray, SPUserField);
+
+        x.done(function (result) {
+            // result is an SP.List because that is what we passed to resolve()!
+            var htmlText = "";
+            for (var i = 0; i < result.users.length; i++) {
+                var user = result.users[i];
+                if (htmlText !== "") { htmlText += "; "; }
+                htmlText += '<a href="/_layouts/15/userdisp.aspx?ID=' + user.get_id().toString() + '&amp;RootFolder=*">' + user.get_title() + '</a>';
```

Comment:
i retrieve user ids and then i put them into href elements. like this:
`'<a href="/_layouts/15/userdisp.aspx?ID=' + user.get_id().toString() + '&amp;RootFolder=*">' + user.get_title() + '</a>'`